hack/release/common.sh

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+config(){
+  GITHUB_ACCOUNT="Azure"
+  RELEASE_ACR=${RELEASE_ACR:-ksnap.azurecr.io} # will always be ovreridden
+  RELEASE_REPO_ACR=${RELEASE_REPO_ACR:-${RELEASE_ACR}/public/aks/karpenter/}
```

Comment:
```suggestion
  RELEASE_REPO_ACR=${RELEASE_REPO_ACR:-${RELEASE_ACR}/public/aks/karpenter}
```

Should the `/` at the end be removed?